src/form/form.tsx

```diff
@@ -400,6 +409,18 @@ class Form<MODEL extends object> extends React.Component<Props<MODEL>, State<MOD
         }, {});
     }
 
+    private getUpdatedFields(prevModel: MODEL, model: MODEL, prevField: string = ''): Array<string> {
+        return flatten(
```

Comment:
Could we test that guy? If necessary just extract this method to a different plan TS file to make it generic and easier to test it.

Reply:
Sure thing, will add tests for this function.

Reply:
@BenjaminEckardt function is extracted and tests are added.
Not sure is It `form` directory good place for this function ?